### ISSUE TITLE
Visual feedback for issues during development

### ISSUE THREAD

**Issue by trentmwillis** [ember-a11y/ember-a11y-testing#5]
After making components aware of accessibility, they should be able to manipulate the visual appearance of an offending element during development to make it obvious something is wrong (turn it red? Big arrow pointing to it? Open to suggestions).

**Comment by noopkat** [comment]
Kahn Academy's [Tota11y](https://khan.github.io/tota11y/) tool does a tremendous job at doing this. I wonder if it could be forked and extended for this test suite's use to include things like elements with an ember action not being focussable etc? It's MIT license which would allow this :smile:

**Comment by trentmwillis** [comment]
This has been implemented (more-or-less) via the violation warning level styles.
